pkg/daemon/drain.go

```diff
@@ -38,16 +43,18 @@ func (dn *Daemon) cordonOrUncordonNode(desired bool) error {
 		err := drain.RunCordonOrUncordon(dn.drainer, dn.node, desired)
 		if err != nil {
 			lastErr = err
-			glog.Infof("cordon/uncordon failed with: %v, retrying", err)
+			glog.Infof("%s failed with: %v, retrying", verb, err)
 			return false, nil
 		}
 		return true, nil
 	}); err != nil {
 		if err == wait.ErrWaitTimeout {
-			return errors.Wrapf(lastErr, "failed to cordon/uncordon node (%d tries): %v", backoff.Steps, err)
+			return errors.Wrapf(lastErr, "failed to %s node (%d tries): %v", verb, backoff.Steps, err)
 		}
-		return errors.Wrap(err, "failed to cordon/uncordon node")
+		return errors.Wrapf(err, "failed to %s node", verb)
 	}
+
+	dn.logSystem("Node has been successfully %s", verb)
```

Comment:
For consistency, I would say let caller decide both logging and sending event.

Reply:
ok, rebased onto master and dropped this on-success log change with f302d7e4 -> 342c0135.